analyzer/analyzer.py

```diff
@@ -323,12 +323,21 @@ def format_result_for_visualizer(self, data):
             for key in sorted(output.keys()):
                 output_sort[node_type][key] = copy.deepcopy( output[key] )
 
-        cpu_core_data_dict = OrderedDict()
-        for key,value in output_sort["ceph"]["cpu"].items():
-            for name,data in value.items():
-                cpu_core_data_dict[name] = data
-        output_sort["ceph"]["cpu"] = cpu_core_data_dict
-
+        for table in output_sort["ceph"].keys():
+            summary_data_dict = OrderedDict()
+            detail_data_dict = OrderedDict()
+            total_data_dict = OrderedDict()
+            if table == "cpu":
+                for key,value in output_sort["ceph"]["cpu"].items():
+                    for name,data in value.items():
+                        if 'all' in name:
+                            summary_data_dict[name] = data
+                        detail_data_dict[name] = data
+            else:
+                summary_data_dict = output_sort["ceph"][table]
+            total_data_dict["summary"] = summary_data_dict
+            total_data_dict["detail"] = detail_data_dict
+            output_sort["ceph"][table] = total_data_dict
```

Comment:
For current design, I don't think you need to add such logic under format_result_for_visualizer(self, data), you can move to the function to process sar data?